File: src/main/java/kookmin/software/capstone2023/timebank/domain/repository/CommentRepository.java
package kookmin.software.capstone2023.timebank.domain.repository;

import kookmin.software.capstone2023.timebank.domain.model.Comment;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CommentRepository extends JpaRepository<Comment, Long> {
    List<Comment> findByInquiryId(Long inquiryId);
    Long countByInquiryId(Long inquiryId);
}

File: src/main/java/kookmin/software/capstone2023/timebank/domain/repository/CommentRepository.kt
package kookmin.software.capstone2023.timebank.domain.repository

import kookmin.software.capstone2023.timebank.domain.model.Comment
import org.springframework.data.jpa.repository.JpaRepository

interface CommentRepository : JpaRepository<Comment, Long> {
    fun findByInquiryId(inquiryId: Long): List<Comment>
    fun countByInquiryId(inquiryId: Long): Long
}
